doc/user-manual/tools/emacs-mode.rst

```diff
@@ -312,7 +312,12 @@ corresponding TeX command:
 - **Subscript** and **superscript**: you can input subscript or
   superscript forms by prepending the character with ``\_`` (subscript)
   or ``\^`` (superscript). Note that not all characters have a
-  subscript or superscript counterpart in Unicode.
+  subscript or superscript counterpart in Unicode. Eg. ``g\_1`` will
```

Comment:
For the sake of logical flow of this text, the "Eg." sentence should come before the "Note that not all..."

Reply:
Yes!

Reply:
Thanks!